posthog/models/filters/mixins/common.py

```diff
@@ -100,7 +100,6 @@ class ClientQueryIdMixin(BaseParamMixin):
     def client_query_id(self) -> Optional[str]:
         return self._data.get(CLIENT_QUERY_ID, None)
 
-    @include_dict
```

Comment:
I'm not yet convinced that this is the correct fix. Could any other code be relying on serializing-and-then-deserializing the filters object?

Also this whole issue indicates that filters hashing is a mess and we probably will have a similar bug later - probably worth taking the time to clean it up, so it is clearer. Perhaps we should just exclude properties prefixed with `_` from hashes?

Also the method is now unused.